test/hotspot/jtreg/vmTestbase/nsk/jdb/fields/fields001/fields001.java

```diff
@@ -55,68 +55,63 @@
+import nsk.share.jdb.JdbTest;
 
-import java.io.*;
-import java.util.*;
+import java.io.PrintStream;
 
 public class fields001 extends JdbTest {
 
-    public static void main (String argv[]) {
+    public static void main(String[] argv) {
         System.exit(run(argv, System.out) + JCK_STATUS_BASE);
     }
 
-    public static int run(String argv[], PrintStream out) {
-        debuggeeClass =  DEBUGGEE_CLASS;
+    public static int run(String[] argv, PrintStream out) {
+        debuggeeClass = DEBUGGEE_CLASS;
         firstBreak = FIRST_BREAK;
         lastBreak = LAST_BREAK;
         return new fields001().runTest(argv, out);
     }
 
-    static final String PACKAGE_NAME       = "nsk.jdb.fields.fields001";
-    static final String TEST_CLASS         = PACKAGE_NAME + ".fields001";
-    static final String DEBUGGEE_CLASS     = TEST_CLASS + "a";
-    static final String DEBUGGEE_CLASS1    = DEBUGGEE_CLASS + "$Inner";
-    static final String DEBUGGEE_CLASS2    = DEBUGGEE_CLASS + "$Extender";
-    static final String FIRST_BREAK        = DEBUGGEE_CLASS + ".main";
-    static final String LAST_BREAK         = DEBUGGEE_CLASS + ".lastBreak";
-    static final String NOT_VALID_SAMPLE   = "is not a valid";
+    static final String PACKAGE_NAME = "nsk.jdb.fields.fields001";
+    static final String TEST_CLASS = PACKAGE_NAME + ".fields001";
+    static final String DEBUGGEE_CLASS = TEST_CLASS + "a";
+    static final String DEBUGGEE_CLASS1 = DEBUGGEE_CLASS + "$Inner";
+    static final String DEBUGGEE_CLASS2 = DEBUGGEE_CLASS + "$Extender";
+    static final String FIRST_BREAK = DEBUGGEE_CLASS + ".main";
+    static final String LAST_BREAK = DEBUGGEE_CLASS + ".lastBreak";
 
     static String[] checkedFields1 = {
-        "i_st",    "o_st",
-        "i_pv",    "o_pv",
-        "i_pt",    "o_pt",
-        "i_pb",    "o_pb",
-        "i_fn",    "o_fn",
-        "i_tr",    "o_tr",
-        "i_vl",    "o_vl",
-        "i_a",     "o_a",
-        "i_aa",    "o_aa",
-        "i_aaa",   "o_aaa"
-                                      };
+            "i_st", "o_st",
+            "i_pv", "o_pv",
+            "i_pt", "o_pt",
+            "i_pb", "o_pb",
+            "i_fn", "o_fn",
+            "i_tr", "o_tr",
+            "i_vl", "o_vl",
+            "i_a", "o_a",
+            "i_aa", "o_aa",
+            "i_aaa", "o_aaa"
+    };
 
     static String[] checkedFields2 = {
-        "ii_pv",    "oi_pv",
-        "ii_pt",    "oi_pt",
-        "ii_pb",    "oi_pb",
-        "ii_fn",    "oi_fn",
-        "ii_tr",    "oi_tr",
-        "ii_vl",    "oi_vl",
-        "ii_a",     "oi_a",
-        "ii_aa",    "oi_aa",
-        "ii_aaa",   "oi_aaa"
-                                      };
+            "ii_pv", "oi_pv",
+            "ii_pt", "oi_pt",
+            "ii_pb", "oi_pb",
+            "ii_fn", "oi_fn",
+            "ii_tr", "oi_tr",
+            "ii_vl", "oi_vl",
+            "ii_a", "oi_a",
+            "ii_aa", "oi_aa",
+            "ii_aaa", "oi_aaa"
+    };
```

Comment:
Why are these indented 8 instead of 4?

Reply:
b/c you get indentation unit for each block,  1st block is `fields001` class definition, next block is the array initializations (start at L#86 and L#99 for `checkedFields1` and `checkedFields2` respecitively)

Reply:
I don't follow. You seem to have added an extra 4 (after having already indented 4 extra spaces) simply because it is an array initialization. Why is the indentation any different than it would be for something like a "for" loop block. The body of the block is indented 4 spaces more than the "for" statement.  For the static initialization the body of that "static" block should be indented 4 more than the "static" statement. I don't see anything in the style guide that would indicate otherwise.

Reply:
oh, sorry, it appears I was trying to explain the new code while looking at the old one...

full disclaimer, which I thought was obvious, the vast majority of the changes here aren't done by me, they are done by auto-formatter, or more precisely by IntelliJ IDEA's formatter. IDEA doesn't consider array initialization as a new block, and what's actually at play here is an indent for statement continuation (which is 2x of regular indent).

Reply:
Do you plan on leaving these indented 8 instead of 4 because that's what IDEA defaults to? Other editors, like emacs, will want to indent them 4, and will do so often do so automatically just by editing the line, depending on whether you type any character that forces reformatting of indentation.

Do we have coding  guidelines that say to indent them 8?

Reply:
> Do we have coding guidelines that say to indent them 8?

old "official" guidelines don't tell much about it,  @aioobe's guidelines don't special-case array initiation, so it is covered by:
> A continuation line should be indented in one of the following four ways
> Variant 1: With 8 extra spaces relative to the indentation of the previous line.
> Variant 2: With 8 extra spaces relative to the starting column of the wrapped expression.
> Variant 3: Aligned with previous sibling expression (as long as it is clear that it’s a continuation line)
> Variant 4: Aligned with previous method call in a chained expression. 

which will suggest that 8 is the right indent here. 

google's code-style says that "array initializer may _optionally_ be formatted as if it were a "block-like construct.", which would mean 4, but note "may" and "_optionally_". personally I'd prefer consistency here, but if you think it will make the lives of emacs-lovers miserable, although I'm sure you can configure emacs to treat array-init as line continuation, and not as a block, I can revert that.

Reply:
I don't consider this to be a continuation line. It just looks too much like a statement block.

My other concern is too much seemingly arbitrary reformatting. I really don't like the idea of reformatting an entire file based on some formatter's notion of what is best or correct, leading to unnecessary reformatting like this one. It muddles the history. The exception would be for a file that is hopelessly formatted incorrectly, like using 2-char indentation instead of 4 in java, or 4 char instead of 2 in C.